### PR TITLE
Coin control fixes

### PR DESCRIPTION
**Pull request by komret**
Coin control bugfixes

## Description
- cropped the SVG in the button
- rounded satoshi amount to prevent displaying incorrectly computed decimals
- changed the property hiding the message form `visibility` to `display` to prevent it from breaking the layout when too long

## Related Issue

#2770 

## Screenshots (if appropriate):

SVG icon before and after:
Screenshot 2022-09-12 at 14 13 44
Screenshot 2022-09-12 at 14 12 33

Rounding before and after:
Screenshot 2022-09-12 at 19 14 02
Screenshot 2022-09-12 at 19 14 38

**QA:** This should fix incorrect computations in the "Missing to your input" message. It happened with some specific numbers in the previous version, e.g. 0.5988788. Please check the rounding is correct.

### CODE REVIEW

File: suite-common/wallet-config/src/networksConfig.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const networks = {
             tx: 'https://btc1.trezor.io/tx/',
             account: 'https://btc1.trezor.io/xpub/',
         },
-        features: ['rbf', 'sign-verify', 'amount-unit'],
+        features: ['rbf', 'sign-verify', 'amount-unit', 'coin-control'],
```
Comment (matejkriz) [inline suite-common/wallet-config/src/networksConfig.ts]: has to be added in taproot features too, otherwise it will not work for taproot accounts

**Comment by komret** [comment]
I removed the other commit, bitcoin-like networks have been tested now, let's support them from the beginning. It would be nice to cherry-pick the icon fix into the release, though.

**Comment by komret** [comment]
I found one more bug related to computation and fixed it in 8ed9197f7958396fe907dacea57682d8a932a711.

**Comment by dahaca** [comment]
Since you are making the changes, could you change the margin between the header and controls here to 20px? 🙏 

image

File: packages/suite/src/views/wallet/send/components/Options/components/BitcoinOptions/components/CoinControl.tsx [hunk 2]
```diff
@@ -105,7 +105,9 @@ export const CoinControl = ({ close }: Props) => {
     const totalOutputs = getTotal(
         outputs.map((_, i) => Number(getDefaultValue(`outputs[${i}].amount`, ''))),
     );
-    const totalOutputsInSats = areSatsUsed ? totalOutputs : totalOutputs * 10 ** network.decimals;
+    const totalOutputsInSats = areSatsUsed
+        ? totalOutputs
+        : Math.round(totalOutputs * 10 ** network.decimals); // round to prevent incorrectly computed decimal, e.g. 0.5988788 produces 59887880.00000001
```
Comment (dahaca) [inline packages/suite/src/views/wallet/send/components/Options/components/BitcoinOptions/components/CoinControl.tsx]: please use `formatAmount()` from `wallet-utils`

**Review by dahaca — CHANGES_REQUESTED** [review]

File: packages/suite/src/views/wallet/send/components/Options/components/BitcoinOptions/components/CoinControl.tsx (hunk 2 above)
Comment (komret) [reply packages/suite/src/views/wallet/send/components/Options/components/BitcoinOptions/components/CoinControl.tsx]: Both changes in 56302c065d02498e4e04b99f2382e9301f2ebf7b.
I think the correct helper to use here is `amountToSatoshi`.

**Review by dahaca — DISMISSED** [review]

**Review by matejkriz — APPROVED** [review]

**Comment by STew790** [comment]
**QA OK**
Icon changed correctly and computation of missing input seems correct as well.

**Info:**
 - Suite version: desktop 22.9.1 (a2cec27a5445c65ca476978302a41b73c9c37994)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/22.9.1 Chrome/100.0.4896.160 Electron/18.3.5 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: model 1 1.11.2 Universal (revision 0d87b55ba4fed7eecc72bf2a94ee473830b095e9)
 - Transport: bridge 2.0.31

**Pull request merged**